src/config.ts

```diff
@@ -9,6 +14,12 @@ export type Config = {
 
   /** Default options for `debug` helper. */
   defaultDebugOptions?: Partial<DebugOptions>;
+
+  /** Option to return composite components */
```

Comment:
```suggestion
  /** Option for byText, byDisplayValue and byPlaceholderText queries to return composite components */
```

Reply:
This will be removed as well once the `allowBreakingChanges` flag is available